pkg/operators/cluster-machine-approver.go

```diff
@@ -0,0 +1,35 @@
+package operators
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/cluster-api-actuator-pkg/pkg/framework"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	cmaDeployment      = "machine-approver"
+	cmaClusterOperator = "machine-approver"
+)
+
+var _ = Describe("[Feature:Operators] Cluster Machine Approver deployment", func() {
+	var client runtimeclient.Client
+	var err error
+	client, err = framework.LoadClient()
```

Comment:
Any reason not to inline the declaration? Or should this be in a before each or something?
```suggestion
	client, err := framework.LoadClient()
```

Reply:
no reason indeed. Thanks!

Reply:
inlined it, let me know if it looks good now.